common/config/rush/common-versions.json

```diff
@@ -37,5 +37,8 @@
     // "typescript": [
     //   "~2.4.0"
     // ]
+    "@cadl-lang/compiler": ["~0.31.0"],
+    "@azure-tools/cadl-autorest": ["^0.22.0"],
+    "ts-node": ["^10.9.1"]
```

Comment:
Forgot to update that one but I think we should be able to use the same one for every package